numba_dpex/experimental/kernel_dispatcher.py

```diff
@@ -177,7 +179,7 @@ class KernelDispatcher(Dispatcher):
 
     """
 
-    targetdescr = dpex_kernel_target
+    targetdescr = dpex_exp_kernel_target
```

Comment:
I can't see usage of `targetdescr`

Reply:
It is needed by the superclass: https://github.com/numba/numba/blob/a4664180ddc91e4c8a37dd06324f4e229a76df91/numba/core/dispatcher.py#L796

The variable is used in the `super().__init__`: https://github.com/numba/numba/blob/a4664180ddc91e4c8a37dd06324f4e229a76df91/numba/core/dispatcher.py#L823